src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = sourceString.split(';');
+
+  const finalResult = {};
+
+  result.forEach((element) => {
+    const [key, value] = element.split(':');
+
+    if (key && value) {
+      finalResult[key.trim()] = value.trim();
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using a `forEach` loop, consider using `reduce` to build your `finalResult` object. This would align with the task's recommendation to avoid loops and use iteration methods instead.